### PR TITLE
GUI-2463: Display alarm status on load balancers landing page

### PR DESCRIPTION
**Pull request by kamalgill**
Implements https://eucalyptus.atlassian.net/browse/GUI-2463

### CODE REVIEW

**Comment by kamalgill** [comment]
Retest this please

File: eucaconsole/static/js/pages/custom_filters.js [hunk 1]
```diff
@@ -50,4 +50,10 @@ angular.module('CustomFilters', [])
         input = input.replace(sub_delims, '');
         return input.toLowerCase();
     };
-});
+})
+.filter('toClassName', function () {
+    return function (input) {
+        return input.replace(' ', '_').toLowerCase();
```
Comment (dkavanagh) [inline eucaconsole/static/js/pages/custom_filters.js]: fwiw, I've seen a number of cases where there is status containing multiple words and a single replace may work for alarm status, but not in the general sense as this filter is intended for. Using a global replace would be better for a generic filter like this:
 input.replace(/ /g, '_').toLowerCase();

Note, my use case replaced a "-" with a space, so the space above may need to be escaped.

Comment (kamalgill) [inline eucaconsole/static/js/pages/custom_filters.js]: @dkavanagh : Good catch -- will fix

File: eucaconsole/static/js/pages/custom_filters.js [hunk 2]
```diff
@@ -53,7 +53,7 @@ angular.module('CustomFilters', [])
 })
 .filter('toClassName', function () {
     return function (input) {
-        return input.replace(' ', '_').toLowerCase();
+        return input.replace(/\s+/g, '_').toLowerCase();
```
Comment (dkavanagh) [inline eucaconsole/static/js/pages/custom_filters.js]: Do you really want to match 1 or more whitespace characters? using /\ /g would be consistent with the intent here.

Comment (kamalgill) [inline eucaconsole/static/js/pages/custom_filters.js]: @dkavanagh: Yeah, I see your point -- we'd probably never have tab chars in the input, for example.  Will simplify.

**Pull request merged**